### DISCUSSION TITLE
Error : Failed to start local tunnel service.

### DISCUSSION THREAD

**Discussion by pallav243** [microsoft/hack-together-teams#45] [Q&A]
Hi,

I am trying to run a conversational bot locally but tunnel service is throwing a 403 Forbidden error.

![image](https://github.com/microsoft/hack-together-teams/assets/109834870/a0b8307f-ea7c-4e55-9b2f-8d5216d19850)

![image](https://github.com/microsoft/hack-together-teams/assets/109834870/fe9ce85c-515b-42e2-bc59-0f830121e5af)

I am using Teams Toolkit v5.0.0  and  Node v18.16.0

Any idea whats wrong and how can I resolve this issue.

Thanks
Pallav

**Comment by XiaofuHuang** [comment]
Thank you for reporting this issue. We are currently investigating the problem.

In the meantime, we recommend trying the following approach: [signing up for a new Microsoft 365 account](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/tools-prerequisites#microsoft-365-developer-program) and use that account to retry the action. If you find that the issue is resolved with the new account, please let us know. This information will greatly assist our investigation, as it suggests that the problem may be isolated to your original account.

**Comment by XiaofuHuang** [comment]
> Hi,
> 
> I am trying to run a conversational bot locally but tunnel service is throwing a 403 Forbidden error.
> 
> ![image](https://user-images.githubusercontent.com/109834870/244287455-a0b8307f-ea7c-4e55-9b2f-8d5216d19850.png)
> 
> ![image](https://user-images.githubusercontent.com/109834870/244287537-fe9ce85c-515b-42e2-bc59-0f830121e5af.png)
> 
> I am using Teams Toolkit v5.0.0 and Node v18.16.0
> 
> Any idea whats wrong and how can I resolve this issue.
> 
> Thanks Pallav

In order to better understand the issue you are facing, could you please provide some additional information?
Have you ever been able to successfully perform the action before this problem occurred? Have you recently switched accounts?
These questions will help us better understand the problem and provide a more accurate solution.

**Reply by daviddqnguyen** [reply]
Any updates?

**Reply by XiaofuHuang** [reply]
Can you try to use dev tunnel CLI to start the dev tunnel?
https://github.com/OfficeDev/TeamsFx/issues/8970#issuecomment-1752462443

**Comment by pallav243** [comment]
I have never been able to successfully start a tunneling service task. Its the first time I am doing it. 

My E5 subscription  account looks like this right now: 

![image](https://github.com/microsoft/hack-together-teams/assets/109834870/5eede659-0259-4fd3-9fd1-47af39267099)

I had created an account but deleted the profile because of the login issue.  Then after a couple of days, I started seeing this second E5 subscription which is active right now.  I dont know if this is causing the tunneling service task to fail.

**Reply by burliEnterprises** [reply]
I am facing the same issue, 403 tunnel cannot be created. Were you able to resolve your issue?
